### DISCUSSION TITLE
گفتمان

### DISCUSSION THREAD

**Discussion by osAIran** [oalee/os-plan-coalition#2] [General]
🙌 درود
⚔ امید وارم حالتون خوب باشه🙃، هرچه زود تر به عدالت برسیم 💪🏼  و فردای بهتری با هم بسازیم 🫱🏾‍🫲🏽. اینجا مکانی برای گفتمان در باره‌ی ائتلاف و این طرح می‌باشد. این یک انجمن و حرکت است که ما با هم می‌سازیم 🤗

**Comment by Jiwe-Mobarez** [comment]
به امید پیروزی

**Comment by osAIran** [comment]
سلام دوستان.
علی هستم ۲۷ ساله متخصص نرم افزار و هوش مصنوعی. ۲۶ سال هم ایران بودم.
هدم من باز شدن دولت، باز متن شدن نرم افزار و باز شدن علم برای ایران هست.
یک طرحی رو هم شروع کردم برای برنامه‌ی بازمتن ائتلاف که هدفم از این کار هم همکاری برای بهتر شدن و روشن شدن و برنامه ریزی برای آینده و این ائتلافه.
برای روشن شدن آینده باید جزئیات برنامه های دولت آینده مشخص بشه. برای همین هدف و کاری که این ائتلاف انجام بده تحقیق علم باز (open science) و تحقیق موردی (case study) روی جهان هست که برنامه های آینده مشخص بشه که مردم تصویر خوبی بتونن از آینده بسازن.
